src/storage/hummock_test/src/failpoint_tests.rs

```diff
@@ -106,9 +116,6 @@ async fn test_failpoints_state_store_read_upload() {
     // sync epoch1 test the read_error
     let ssts = hummock_storage.sync(1).await.unwrap().uncommitted_ssts;
     meta_client.commit_epoch(1, ssts).await.unwrap();
-    local_version_manager
-        .refresh_version(meta_client.as_ref())
```

Comment:
Need wait_epoch as a replacement of refresh_version too.

Reply:
> Need wait_epoch as a replacement of refresh_version too.

in previous commit I replaced. I just reset it for coverage test. now it has been resumed.